source/client/app/NinecraftApp.cpp

```diff
@@ -57,7 +57,7 @@ void NinecraftApp::initGLStates()
 	glCullFace(GL_BACK);
 	glEnable(GL_TEXTURE_2D);
 	glHint(GL_PERSPECTIVE_CORRECTION_HINT, GL_FASTEST);
-	glDisable(GL_LIGHTING);
+	//glDisable(GL_LIGHTING);
```

Comment:
Why?

Reply:
I removed support for GL lighting in my GLES compatibility layer because literally nothing was using it and removing it simplified the code (and shaders) a lot. In this case, `GL_LIGHTING` is disabled a lot but never actually enabled or configured.

I can easily add it back if anything actually needs it.

Reply:
I suppose just disable it once in `initGLStates` then, in the case that the OpenGL version provided does support lighting.

Reply:
But why? It's never enabled and disabled by default

Reply:
Great! Then this is resolved

For reference:
> To enable and disable lighting calculation, call [glEnable](https://registry.khronos.org/OpenGL-Refpages/gl2.1/xhtml/glEnable.xml) and [glDisable](https://registry.khronos.org/OpenGL-Refpages/gl2.1/xhtml/glDisable.xml) with argument GL_LIGHTING. Lighting is initially disabled.